src/sass/base/dsf-fonts.scss

```diff
@@ -1,3 +1,116 @@
 // -----------------------------------------------------------------------------
 // This file contains all @font-face declarations, if any.
-// -----------------------------------------------------------------------------
\ No newline at end of file
+// -----------------------------------------------------------------------------
+
+// Roboto Thin
+@font-face {
+  font-family: "Roboto";
+  font-style: normal;
+  font-display: swap;
+  src: url("../../../build/fonts/Roboto-Thin.ttf") format("truetype");
+  font-weight: 100;
+}
+
+ // Roboto Thin Italic
+ @font-face {
+  font-family: "Roboto";
+  font-style: italic;
+  font-display: swap;
+  src: url("../../../build/fonts/Roboto-ThinItalic.ttf") format("truetype");
+  font-weight: 100;
+}
+
+// Roboto Light
+@font-face {
+  font-family: "Roboto";
+  font-style: normal;
+  font-display: swap;
+  src: url("../../../build/fonts/Roboto-Light.ttf") format("truetype");
+  font-weight: 300;
+}
+
+// Roboto Light Italic
+@font-face {
+  font-family: "Roboto";
+  font-style: italic;
+  font-display: swap;
+  src: url("../../../build/fonts/Roboto-LightItalic.ttf") format("truetype");
+  font-weight: 300;
+}
+
+// Roboto Regular
+@font-face {
+    font-family: "Roboto";
+    font-style: normal;
+    font-display: swap;
+    src: url("../../../build/fonts/Roboto-Regular.ttf") format("truetype");
+    font-weight: 400;
+}
+
+// Roboto Italic
+@font-face {
+  font-family: "Roboto";
+  font-style: italic;
+  font-display: swap;
+  src: url("../../../build/fonts/Roboto-Italic.ttf") format("truetype");
+  font-weight: 400;
+}
+
+// Roboto Medium
+@font-face {
+  font-family: "Roboto";
+  font-style: normal;
+  font-display: swap;
+  src: url("../../../build/fonts/Roboto-Medium.ttf") format("truetype");
+  font-weight: 500;
+}
+
+// Roboto Medium Italic
+@font-face {
+  font-family: "Roboto";
+  font-style: italic;
+  font-display: swap;
+  src: url("../../../build/fonts/Roboto-MediumItalic.ttf") format("truetype");
+  font-weight: 500;
+}
+
+// Roboto Bold
+@font-face {
+  font-family: "Roboto";
+  font-style: normal;
+  font-display: swap;
+  src: url("../../../build/fonts/Roboto-Bold.ttf") format("truetype");
+  font-weight: 700;
+}
+
+// Roboto Bold Italic
+@font-face {
+  font-family: "Roboto";
+  font-style: italic;
+  font-display: swap;
+  src: url("../../../build/fonts/Roboto-BoldItalic.ttf") format("truetype");
+  font-weight: 700;
+}
+
+// Roboto Black
+@font-face {
+  font-family: "Roboto";
+  font-style: normal;
+  font-display: swap;
+  src: url("../../../build/fonts/Roboto-Black.ttf") format("truetype");
+  font-weight: 900;
+}
+
+// Roboto Black Italic
+@font-face {
+  font-family: "Roboto";
```

Comment:
I think font-family name needs to be unique when setting multiple font-faces
https://stackoverflow.com/a/4872624

Reply:
fixed